src/Sentry.Unity/SentryUnity.cs

```diff
@@ -13,6 +15,9 @@ namespace Sentry.Unity
     /// </summary>
     public static class SentryUnity
```

Comment:
Should this be `IDisposable` too?

Reply:
I'm not sure how that would work right now or what we would dispose. It can't be the `SentryUnity` class because it'd have to no longer be `static`. So the `Init` method should return some intermediate `IDisposable`.